### ISSUE TITLE
Implement golang error inteface

### ISSUE THREAD

**Issue by Nightapes** [exasol/error-reporting-go#7]
For easy use in golang, it would be good to implement the native error interface. With the error interface, the ExaError can be used as a normal golang error 

```
type error interface {
    Error() string
}```

I can open an PR if you want

**Comment by AnastasiiaSergienko** [comment]
Nice idea, @Nightapes! Please feel free to open a pull request

**Comment by AnastasiiaSergienko** [comment]
Closed with #8
